test/minting2/MintControllerTests.js

```diff
@@ -136,7 +137,7 @@ async function run_tests(newToken, accounts) {
         await checkMINTp0([token, mintController], [expectedTokenState, expectedMintControllerState]);
 
         // increment minter allowance
-        await expectRevert(mintController.incrementMinterAllowance(amount, {from: Accounts.controller1Account}));
+        await expectError(mintController.incrementMinterAllowance(amount, {from: Accounts.controller1Account}), "Can only increment allowance for enabled minter");
```

Comment:
I think there might be a bug with our expectError method because in this line we are expecting to find a revert string "Can only increment allowance for enabled minter" but that is no longer in the contract. Yet the tests are still passing, so this looks like the expectError function is not checking the expected revert string properly.